### PR TITLE
add memcached pkg & its deps

### PR DESCRIPTION
**Pull request by tuananh**
Signed-off-by: Tuan Anh Tran <me@tuananh.org>

### CODE REVIEW

**Review by dlorenc — DISMISSED** [review]
Nice!

File: Makefile [hunk 1]
```diff
@@ -281,5 +281,10 @@ $(eval $(call build-package,dav1d,1.0.0-r0))
 $(eval $(call build-package,aom,3.5.0-r0))
 $(eval $(call build-package,libavif,0.11.1-r0))
 $(eval $(call build-package,gd,2.3.3-r0))
+$(eval $(call build-package,py3-sphinx,6.0.0-r0))
+$(eval $(call build-package,heimdal,7.8.0-r0))
+$(eval $(call build-package,cyrus-sasl,2.1.28-r0))
+$(eval $(call build-package,py3-sphinx,6.0.0-r0))
```
Comment (luhring) [inline Makefile:287]: We have the line `$(eval $(call build-package,py3-sphinx,6.0.0-r0))` twice here

**Review by luhring — DISMISSED** [review]

**Comment by dlorenc** [comment]
Whoops, sent https://github.com/wolfi-dev/os/pull/351

**Comment by dlorenc** [comment]
I fixed the linter but it looks like you'll have to rebase to pick it up.

**Review by dlorenc — APPROVED** [review]

**Review by luhring — APPROVED** [review]

**Pull request merged**